test/performance/event_handler_test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * This script simulates the creation of new pods in the cluster and measures
+ * the number of events processed by YHS.
+ * It calculates and logs the total number of pods created
+ * and the total number of events processed by YHS during the test.
+ */
+import { Kubernetes } from 'k6/x/kubernetes';
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+const namespace = __ENV.NAMESPACE || 'default';
+const yhsUrl = "http://localhost:8989";
```

Comment:
extract the url into an envvar, we might target remote environments, or we might use a different port